### DISCUSSION TITLE
Classic vanilla look for geeks :c

### DISCUSSION THREAD

**Discussion by arno-arno** [plugdata-team/plugdata#243] [Ideas]
Yeah, I mean... **curvy cords, nice inlets** and **outlets** look cool and fluid, but I personally have a problem with making spiders-like patches. Max-like look makes less space for objects and confuses the eye which has to change direction few times while getting from object to object. This is really good for educational purposes, but for a vanilla user like me this makes very hard to efficient use.

I would like to see if there is some more users who would like classic vanilla look in PlugData. It could be solved by adding more options to gui options tab. Also as a beginner programer would like to contribute in this.

**Comment by timothyschoen** [comment]
You can come a long way by setting all the colours to be monochrome, but adding a few more options like that would be okay with me.

<img width="1054" alt="Screenshot 2022-11-26 at 20 12 01" src="https://user-images.githubusercontent.com/44585538/204105364-91b7244f-4099-41e1-b046-ff6445e0ea62.png">

**Comment by timothyschoen** [comment]
I've had more people request an option for non-curved patch chords, you could start with that if you want. 

If you need help getting started, let me know :)

**Comment by AlexMitchellMus** [comment]
Implementing this not only honors history, but also helps to fine-tune the integration of nodes and cables.

**Comment by timothyschoen** [comment]
<img width="1046" alt="Screenshot_2023-01-20_at_17 09 53" src="https://user-images.githubusercontent.com/44585538/213749012-bbe75f36-f9a5-4c5b-aa1c-a18aa5b8e88e.png">

I'd say we pretty much there with the latest changes on develop!
